src/components/ListFilter/ListFilter.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { FilterType } from '../../enums/FilterType';
+
+type Props = {
+  todosCount: number;
+  hasCompletedTodos: boolean;
+  filterType: FilterType;
+  onFilterTypeChange: (newType: FilterType) => void;
+};
+
+export const ListFilter: React.FC<Props> = ({
+  todosCount,
+  hasCompletedTodos,
+  filterType,
+  onFilterTypeChange,
+}) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {`${todosCount} items left`}
+    </span>
+
+    <nav className="filter">
+      {(Object.keys(FilterType) as (keyof typeof FilterType)[]).map(
+        typeOfFilter => {
+          const type = FilterType[typeOfFilter];
+
+          return (
+            <a
+              key={type}
+              href={type !== 'all'
+                ? `#/${type}`
+                : '#/'}
+              className={classNames(
+                'filter__link',
+                { selected: filterType === type },
+              )}
+              onClick={() => onFilterTypeChange(type)}
+            >
+              {type.charAt(0).toUpperCase() + type.slice(1)}
+            </a>
+          );
+        },
+      )}
+    </nav>
+
+    {hasCompletedTodos && (
+      <button
+        type="button"
+        className="todoapp__clear-completed"
+      >
+        Clear completed
+      </button>
+    )}
```

Comment:
it is not correct because your filter buttons should stay in the middle of app like here 
![Screenshot 2023-04-10 at 14 47 26](https://user-images.githubusercontent.com/91826635/230896044-c82dd64d-e609-4ad4-9767-47d716bad9e2.png)

```suggestion
      <button
        type="button"
        className={classNames('todoapp__clear-completed', {
          'is-invisible': todos.every(todo => !todo.completed),
        })}
      >
        Clear completed
      </button>
```

Reply:
and also you don't need now hasCompletedTodos constant